homeassistant/components/iskra/entity.py

```diff
@@ -0,0 +1,63 @@
+"""Base entity for Iskra devices."""
+
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.entity import Entity
+
+from .const import DOMAIN, MANUFACTURER
+
+
+class IskraDevice(Entity):
```

Comment:
Shouldn't this inherit the coordinator entity?

Reply:
i changed both IskraEntity and IskraSensor. I hope now they are structured ok